### PR TITLE
three count

### PR DESCRIPTION
**Pull request by chevalier-88**

### CODE REVIEW

File: homework0/vladimir_chevalier/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+hello hellel
```
Comment (Dmitry404) [inline homework0/vladimir_chevalier/readme.md]: The school's name should read `hillel` :)

Please change folder's name to fit the `name.surname` pattern (lower case, dot separated)

Comment (chevalier-88) [reply homework0/vladimir_chevalier/readme.md]: Sorry for my mistake (this is why I long time could not connected to wi fi in the school) ))))))
I fix mistake. But I have question, I need make "fork" for directory "homework" or for each new directory ("homework0","homework1"...) ?

**Comment by chevalier-88** [comment]
try add my example...

File: homework0/vladimir_chevalier/readme.md (hunk 1 above)
Comment (Dmitry404) [reply homework0/vladimir_chevalier/readme.md]: To fork once is enough. Then you will need to sync your repo with "upstream". I'm going to explain how to do that in a later lessons. But if you would like to do it yourself, here is short [instruction](https://help.github.com/articles/syncing-a-fork).

Comment (Dmitry404) [reply homework0/vladimir_chevalier/readme.md]: Don't worry, that's fine. Just practice more. 

As we all contribute to same repository, you get homework which belong to others. It's okay.

**Pull request merged**